packages/generate-password/src/index.ts

```diff
@@ -0,0 +1,62 @@
+type CharacterPoolOptions = {
```

Comment:
I'm by no means an expert in that area. I did that add that but changed it because I wasn't sure about what the exact benefit of using `Partial` is in this context. https://github.com/Automattic/wp-calypso/pull/71704/commits/83a7401f17bd597ac69fc06df5ffb74102f76524 brings it back.

Reply:
It basically does the job of setting all fields to optional, rather than having you to do that yourself manually. It also sets the expectations to us about what to expect from that type (that each of its fields are optional) without having to parse the entire type.

Reply:
Alright. I just saw it from another angle but it works well so it's fine by me 👍🏻